pkg/operators/wasm/maps.go

```diff
@@ -122,6 +122,7 @@ func (i *wasmOperatorInstance) newMap(ctx context.Context, m wapi.Module, stack
 }
 
 // getMap gets an existing map.
+// releaseHandle must be called when the map is no longer needed.
```

Comment:
It could be updated, but nothing checks against it.
The modification to check the handles from https://github.com/inspektor-gadget/inspektor-gadget/pull/3758 is needed, but that PR is based on this one